src/preamble.js

```diff
@@ -631,11 +642,11 @@ function stringToUTF8Array(str, outU8Array, outIdx, maxBytesToWrite) {
 // Use the function lengthBytesUTF8() to compute the exact number of bytes (excluding null terminator) that this function will write.
 // Returns the number of bytes written, EXCLUDING the null terminator.
 
-function stringToUTF8(str, outPtr, maxBytesToWrite) {
+function stringToUTF8(str, outPtr, maxBytesToWrite, withNullTermination) {
```

Comment:
The comment for this function still unconditionally states "null-terminated". Would be good to update these to match "optionally null-terminated".